app/src/api.ts

```diff
@@ -29,9 +29,11 @@ export const onRequest = (config: InternalAxiosRequestConfig): Promise<InternalA
 		if (config.url && config.url === '/auth/refresh') {
 			queue.pause();
 			resolve(requestConfig);
-			queue.start();
 		} else {
-			queue.add(() => resolve(requestConfig));
+			queue.add(() => {
+				requestConfig.headers['Authorization'] = api.defaults.headers.common['Authorization'];
+				return resolve(requestConfig);
+			});
```

Comment:
Teeny-tiny question / suggestion:

Could we return in the first `if` case and get rid of the `else`?

Reply:
Makes sense, made the change 👍